AK/UFixedBigInt.h

```diff
@@ -0,0 +1,771 @@
+/*
+ * Copyright (c) 2021, Leon Albrecht <leon2002.la@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Checked.h>
+#include <AK/Concepts.h>
+#include <AK/Format.h>
+#include <AK/NumericLimits.h>
+#include <AK/StdLibExtraDetails.h>
+#include <AK/StdLibExtras.h>
+#include <AK/String.h>
+#include <AK/StringBuilder.h>
+
+namespace AK {
+
+template<typename T>
+requires(sizeof(T) >= sizeof(u64) && IsUnsigned<T>) class UFixedBigInt;
+
+// FIXME: This breaks formatting
+// template<typename T>
+// constexpr bool Detail::IsIntegral<UFixedBigInt<T>> = true;
+
+template<typename T>
+constexpr bool Detail::IsUnsigned<UFixedBigInt<T>> = true;
```

Comment:
Please make these `inline`, including this header twice in the same target _can_ generate link errors.

Reply:
I did not know that, thanks for the hint